atomicapp/providers/openshift.py

```diff
@@ -36,91 +35,205 @@ class OpenShiftProvider(Provider):
     cli = None
     config_file = None
     template_data = None
+    providerapi = "https://127.0.0.1:8443"
+    openshift_api = None
+    kubernetes_api = None
```

Comment:
Why is kubernetes referenced here? I know it's a base of OpenShift, but why would we ever wanted to use it in OS provider?